app/components/list-uploaded-assets/ListUploadedAssets.tsx

```
@@ -0,0 +1,95 @@
+      }
+    };
+
+    fetchUploadedAssets();
+  }, [props]);
+
+  const filteredCreatorAssets: Asset[] = useMemo(() => {
+    return Array.isArray(assets)
+      ? assets.filter(
+          (ast: Asset) =>
+            ast.creatorId &&
+            ast.creatorId.value.toLowerCase() ==
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)